docs/docs/learn/security/privileged-roles.md

```diff
@@ -0,0 +1,172 @@
+- Invalid state proposals can be used to execute invalid withdrawals after 7 days.
+
+### Mitigations
+
+- Compromised proposer address can be replaced by the [L1 Proxy Admin](#l1-proxy-admin).
+- Invalid state proposals can be challenged by the [Challenger](#challenger) within 7 days.
+
+### Address
+
+- **Ethereum**: [`0x7cB1022D30b9860C36b243E7B181A1d46f618C69`](https://etherscan.io/address/0x7cB1022D30b9860C36b243E7B181A1d46f618C69)
+
+## Challenger
+
+### Description
+
+The Challenger is an address that can be used to challenge invalid state proposals submitted by the [Proposer](#proposer) role.
+
+### Risks
+
+- Compromised challenger could invalidate valid state proposals.
+- Compromised challenger could fail to challenge invalid state proposals.
+
+### Mitigations
+
+- Compromised challenger address can be replaced by the [L1 Proxy Admin](#l1-proxy-admin).
+- Challenges can be executed by replaced challenger address.
+- Challenger is a 4-of-6 [multisig](https://etherscan.io/address/0xC91482A96e9c2A104d9298D1980eCCf8C4dc764E#readProxyContract).
+
+### Address
+
+- **Ethereum**: [`0xC91482A96e9c2A104d9298D1980eCCf8C4dc764E`](https://etherscan.io/address/0xC91482A96e9c2A104d9298D1980eCCf8C4dc764E#readProxyContract)
+
+## Guardian
+
+### Description
+
+The Guardian is an address that can be used to pause withdrawals from BOB.
+This is a backup safety mechanism that allows for a temporary halt in the event of a security concern.
+The Guardian role cannot pause specific withdrawals and can only pause all withdrawals.
+
+### Risks
+
+- Compromised guardian could pause withdrawals indefinitely.
+
+### Mitigations
+
+- Compromised guardian address can be replaced by the [L1 Proxy Admin](#l1-proxy-admin).
+- Withdrawals can be unpaused by replaced guardian address.
+- Guardian is a 4-of-6 [multisig](https://etherscan.io/address/0xC91482A96e9c2A104d9298D1980eCCf8C4dc764E#readProxyContract).
+
+### Address
+
+- **Ethereum**: [`0xC91482A96e9c2A104d9298D1980eCCf8C4dc764E`](https://etherscan.io/address/0xC91482A96e9c2A104d9298D1980eCCf8C4dc764E#readProxyContract)
+
+## ERC20 Contract Upgrade Proxy
+
+### Description
+
+The ERC20 Contract Upgrade Proxy is an address that can be used to upgrade four ERC20 contracts on BOB to new versions: USDC, tBTC, wstETH, and STONE. This is a temporary measure:
+
+- USDC: The ERC20 is upgradable to allow Circle to take over the contract to enable native minting and redeeming on BOB.
+- tBTC: The ERC20 is upgradable to allow Threshold governance to take over the contract to enable native minting and redeeming on BOB.
+- wstETH: The ERC20 is upgradable to allow Lido governance to take over the contract to control the contract from the Ethereum side.
+- STONE: The ERC20 is upgradable to allow the Stone team to take over the contract and replace the OP standard bridge contracts with LayerZero.
+
+All other ERC20 contracts on BOB are not upgradable by this proxy.
```

Comment:
(e)sov and (e)dllr also behind proxy

Reply:
Don't think we need to mention that on our side since these will be handled by Sovryn